pkg/antctl/transform/networkpolicy/response.go

```diff
@@ -181,20 +182,27 @@ func EvaluationTransform(reader io.Reader, _ bool, _ map[string]string) (interfa
 var _ common.TableOutput = new(EvaluationResponse)
 
 func (r EvaluationResponse) GetTableHeader() []string {
-	return []string{"NAME", "NAMESPACE", "POLICY-TYPE", "RULE-INDEX", "DIRECTION"}
+	return []string{"NAME", "NAMESPACE", "POLICY-TYPE", "RULE-INDEX", "DIRECTION", "ACTION"}
 }
 
 func (r EvaluationResponse) GetTableRow(_ int) []string {
 	if r.NetworkPolicyEvaluation != nil && r.Response != nil {
+		action := "Allow"
+		if r.Response.Rule.Action != nil {
+			action = string(*r.Response.Rule.Action)
+		} else if r.Response.RuleIndex == math.MaxInt32 {
+			action = "Isolation"
```

Comment:
I think @tnqn suggested `Isolate` for this one (to stick to a verb), which I also prefer

Reply:
Sure, I misunderstood.

Reply:
Changed to "Isolate"